code/main.py

```diff
@@ -1,191 +1,76 @@
-        replace_empty_with="G" if empty_answers_as_g else "")
-    answers_results.save(output_folder,
-                         "results",
-                         sort_results,
-                         timestamp=files_timestamp)
-
-    success_string = "✔️ All exams processed and saved.\n"
-
-    if keys_file:
-        keys_results.add_file(keys_file)
-
-    if (keys_results.row_count == 0):
-        success_string += "No exam keys were found, so no scoring was performed."
-    elif (arrangement_file and keys_results.row_count == 1):
-        answers_results.reorder(arrangement_file)
-        keys_results.data[1][keys_results.field_columns.index(
-            grid_i.Field.TEST_FORM_CODE)] = ""
-
-        answers_results.save(output_folder,
-                             "rearranged_results",
-                             sort_results,
-                             timestamp=files_timestamp)
-        success_string += "✔️ Results rearranged based on arrangement file.\n"
-
-        keys_results.delete_field_column(grid_i.Field.TEST_FORM_CODE)
-        keys_results.save(output_folder,
-                          "key",
-                          sort_results,
-                          timestamp=files_timestamp,
-                          transpose=True)
-
-        success_string += "✔️ Key processed and saved.\n"
-
-        scores = scoring.score_results(answers_results, keys_results,
-                                       form_variant.num_questions)
-        scores.save(output_folder,
-                    "rearranged_scores",
-                    sort_results,
-                    timestamp=files_timestamp)
-        success_string += "✔️ Scored results processed and saved."
-    elif (arrangement_file):
-        success_string += "❌ Arrangement file and keys were ignored because more than one key was found."
-    else:
-        keys_results.save(output_folder,
-                          "keys",
-                          sort_results,
-                          timestamp=files_timestamp)
-        success_string += "✔️ All keys processed and saved.\n"
-        scores = scoring.score_results(answers_results, keys_results,
-                                       form_variant.num_questions)
-        scores.save(output_folder,
-                    "scores",
-                    sort_results,
-                    timestamp=files_timestamp)
-        success_string += "✔️ All scored results processed and saved."
-
-    if (output_mcta):
-        transform_and_save_mcta_output(answers_results, keys_results, files_timestamp, output_folder)
-
-    progress.set_status(success_string, False)
-except (RuntimeError, ValueError) as e:
-    wrapped_err = "\n".join(textwrap.wrap(str(e), 70))
-    progress.set_status(f"Error: {wrapped_err}", False)
-    if debug_mode_on:
-        raise
-progress.show_exit_button_and_wait()
+from process_input import process_input
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser('OMCR',
```

Comment:
The `'OMCR'` name parameter defines our program name as OMCR, which hints to users that it can be called from the terminal like `OMCR ...`. However, it actually can only be called with `main.py ...`. If you just remove the argument, argparser can automatically infer the program name correctly, so I think we should just remove it.